gazebo_ros2_control/README.md

```diff
@@ -0,0 +1,7 @@
+# Gazebo ros_control Interfaces
+
+This is a ROS 2 package for integrating the `ros_control` controller architecture
```

Comment:
I think the package is now called `ros2_control`. We could also link to it: https://github.com/ros-controls/ros2_control